ui/src/components/utils/DateUtils.js

```diff
@@ -36,6 +36,15 @@ class DateUtils {
         return this.getLocalDate(currDt, currentTimezone, 'UTC');
     }
 
+    getDateMinusSevenDays(currentDate, currentTimezone) {
+        debugger;
+        let currDt = new Date();
```

Comment:
currentDate is also taken as a parameter. need to remove either one

Reply:
it is refreshdate and not current date